docs/glossary.md

```diff
@@ -0,0 +1,184 @@
+---
+id: glossary
+title: Glossary
+sidebar_label: Glossary
+description:  Avail common reference for general definitions and concepts.
+
+keywords:
+    - docs
+    - avail
+    - modularity
+    - Monolithic
+    - data availability
+image: https://availproject.github.io/img/avail/AvailDocs.png
+
+
+
+---
+
+# Avail Glossary
+## General
+
+### Modular Blockchain
+A modular blockchain is one that focuses on handling a select few duties and outsources the rest to one or more separate layers.
+
+### Monolithic 
+In a monolithic chain, all tasks are handled on a single layer or a group of tightly coupled chains operating on the same layer. The latter distinction is important to note: a network of interconnected blockchains that handle all roles, such as Polkadot’s parachains or Avalanche’s subnets, doesn’t qualify as modular architecture.
+
+### Data Availability
+Data availability is the guarantee that block proposers publish all transaction data of a block, along with making this data available to all network participants. In essence, it refers to all nodes’ ability to access and download all the data contained within the various blocks of a network.
+
+### Scalability
+Blockchain scalability refers to a blockchain's capacity to execute transactions, store data, and come to consensus as the network's user base grows.
+
+### Execution
+Execution is how nodes on the blockchain process transactions to transition the blockchain between states. Nodes participating in consensus must execute transactions using their copy of the blockchain to attest before validating blocks.
+
+### Settlement
+The "finality" that blockchains offer is a promise that the transactions that have been recorded in the chain's history are unchangeable (or "immutable"). The blockchain must be persuaded of a transaction's authenticity for this to occur. The chain must therefore validate transactions, check the accuracy of proofs, and resolve disagreements in order to perform the settlement function.
+
+### Decoupling
+The concept behind a modular blockchain is that it can focus on a few tasks rather than attempting to perform all of them. In particular, modular blockchains decouple execution from consensus.
+
+### Bridging
+Connection between different layers in modular blockchain
```

Comment:
This isn't correct. Some interactions between modular layers involve bridging, others not. Bridging is a somewhat complex topic, I think you need to do more research before writing an answer.

At a minimum, I'd want this answer to touch on attestation bridging, what we intend to provide at launch with Avail, and bridging other things, like tokens and data in general. It would also be good to talk about bridging between Avail rollups, or at least mention the possibility.